src/site/markdown/getting_started.md

```diff
@@ -0,0 +1,5 @@
+# Getting Started
```

Comment:
How does this content then relate to the `README.md` on the site now? We'd still need something there, and I don't want to duplicate too much content. Although I kind of like the `README.md` as a self-contained one-pager about the project, it could be slimmed down further to the essentials, and port most of the content to the site.